tests/payment/models/Payment - Credit card - Redirect.model.js

```diff
@@ -0,0 +1,75 @@
+import { setup, visitAndClearCookies } from '../../../utils'
+import {
+  fillEmail,
+  getRandomEmail,
+  fillProfile,
+} from '../../../utils/profile-actions'
+import {
+  goToPayment,
+  fillShippingInformation,
+} from '../../../utils/shipping-actions'
+import {
+  completePurchase,
+  confirmRedirect,
+  payWithCreditCard,
+} from '../../../utils/payment-actions'
+import { ACCOUNT_NAMES } from '../../../utils/constants'
+
+export default function test(account) {
+  describe(`Payment - Credit Card  - Two cards - ${account}`, () => {
+    before(() => {
+      visitAndClearCookies(account)
+    })
+
+    it('with only delivery', () => {
+      const email = getRandomEmail()
+
+      setup({ skus: ['289'], account })
+      fillEmail(email)
+      fillProfile({
+        lastName: 'Redirect',
+      })
+      fillShippingInformation(account)
+      if (account === ACCOUNT_NAMES.NO_LEAN) {
+        cy.get('#shipping-data')
+          .contains('PAC')
+          .should('be.visible')
+        cy.get('#shipping-data')
+          .contains('Motoboy')
+          .should('be.visible')
+        cy.get('#shipping-data')
+          .contains('Expressa')
+          .should('be.visible')
+        cy.get('#shipping-data')
+          .contains('PAC Lento')
+          .should('be.visible')
+      } else {
+        cy.get('#shipping-data')
+          .contains('Mais rápida')
```

Comment:
Is there any plan for the future to remove this kind of verification? Does not seem very safe to apply a test based on a language string.

Reply:
No. We can avoid on new Checkout on IO, but I don't think we will do that for v6 tests.
We run these tests on specific accounts with the precise configurations we need, so for now it's not an issue.

Reply:
Got it. Yes, for the IO version it would be good to be language agnostic.